core/db/utils.py

```
@@ -0,0 +1,25 @@
+from typing import AsyncGenerator
+from fastapi import Depends
```

Ruff (0.8.2)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)